.tests/integration/config.yaml

```diff
@@ -5,9 +5,9 @@ tools:
   common: "docker://marrip/common:1.1.0"
   default: "docker://gmsuppsala/somatic:develop"
 
- references:
-   genome: "/data/ref_genomes/hg19/bwa/BWA_0.7.10_refseq/hg19.with.mt.fasta"
+references:
+  genome: "/data/ref_genomes/hg19/bwa/BWA_0.7.10_refseq/hg19.with.mt.fasta"
 
- bwa-mem:
-   threads: 2
-   extra: r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample} -v 1 '"
+bwa_mem:
+  threads: 2
+  extra: r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample} -v 1 '"
```

Comment:
Just thinking that maybe we should have `params: extra:` to follow the snakemake rule fields?

Reply:
I agree.
Can you have that on one row like this:
bwa_mem:
  params: extra: r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample} -v 1 '"

Reply:
I would probably do:

```
params:
  extra: ...
```

In case there are several params we want to set through the `config.yaml`.